README.md

```diff
@@ -166,7 +166,7 @@ every field, only overriding those we need to.
 
 ## Contributing
 
-Thumbprint accepts issues and pull requests. Take at look at our [contribution instructions](CONTRIBUTING.md) if you'd like to contribute.
+Kotlin Testing Tools accepts issues and pull requests. Take at look at our [contribution instructions](CONTRIBUTING.md) if you'd like to contribute.
```

Comment:
We might want to actually name our company here. Like say "Thumbtack, the maintainer of the Kotlin Testing Tools repository,". But fine with this for now.